src/core/namespace.c

```diff
@@ -2091,6 +2127,9 @@ int setup_namespace(
         }
 
         if (n_extension_images > 0 || !strv_isempty(extension_directories)) {
+                // r = parse_env_extension_hierarchies(&hierarchies, image_class_info[dissected_image->image_class].name_env);
```

Comment:
This is the part that I'm confused about @bluca. What is wrong with this commented out line to set up the overlay acc to sysext or confext.

I wanted to replace that commented out line with the one that is currently uncommented:
(r = parse_env_extension_hierarchies(&hierarchies,"SYSTEMD_SYSEXT_HIERARCHIES");)
but unsure why that causes the testsuite to crash..

Reply:
It cannot be either or, it needs to be both, as it can have both types at the same time